### ISSUE TITLE
Welcome: NVDA reads some irrelevant content before reading the information of lightbox on hitting "Interface Overview" button

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51983]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Navigate to Welcome screen.
3. Navigate to "Interface Overview" button and hit "Enter".
4. Use down arrow key to read the content that pops up.

**Actual:**
NVDA reads some irrelevant content before reading the information of lightbox. It reads the "Interactive Playground" button,information of status bar and then reads the content of the lightbox.

**Same issue has been observed with JAWS18.**

**Expected:**
NVDA should only read the content of the lightbox once it appears. Irrelevant content should not be read before reading the content.

**Recommendations:**
Provide tabindex to the entire lightbox that has appeared and give title to it. Then the content of the lightbox should be read.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not know the exact content that has been presented using lightbox and also user will not know on which screen he is on.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_WelcomeScreen_ScreenReader_LightboxIssue.pptx](https://github.com/Microsoft/vscode/files/2105350/A11y_VSCode_WelcomeScreen_ScreenReader_LightboxIssue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Same as the other welcome issue, assigning to October so we see what we do here.
